src/config/createWebpackConfig.js

```diff
@@ -203,12 +204,26 @@ module.exports = (target = 'node', options) => {
   }
   if (WEB_PROD) {
     config.optimization = {
+      runtimeChunk: true,
       splitChunks: {
+        cacheGroups: {
+          commons: {
+            test: /[\\/]node_modules[\\/]/,
+            name: 'vendor',
+            chunks: 'all'
+          }
+        },
         chunks: 'all',
         name: false
       }
     }
   }
+  config.plugins.push(
+    new CleanPlugin(['.tapestry'], {
```

Comment:
Will this clean it for both `server` and `client`? Does that matter? When I thought about this earlier I assumed the server would fill `.tapestry` then the client webpack compiler would clean it again before adding the client files.

Reply:
It seems that it only cleans the files that's webpack's `output: {}` -ing at that time. Or I've got something weird going with my setup and it's not actually working...

Reply:
oh cool, that's pretty spesh.